cdc/sink/mq_flush_worker.go

```diff
@@ -177,7 +186,17 @@ func (w *flushWorker) asyncSend(
 
 // run starts a loop that keeps collecting, sorting and sending messages
 // until it encounters an error or is interrupted.
-func (w *flushWorker) run(ctx context.Context) error {
+func (w *flushWorker) run(ctx context.Context) (retErr error) {
+	defer func() {
+		select {
+		case w.errCh <- retErr:
+		default:
+			log.L().Warn("flushWorker error is dropped due to a previous erro", zap.Error(retErr))
```

Comment:
how about follow the log calling convention, just call `Warn` directly ?

Reply:
my mistake